eng/helix/content/runtests.cmd

```diff
@@ -4,35 +4,36 @@ setlocal enabledelayedexpansion
 
 REM Use '$' as a variable name prefix to avoid MSBuild variable collisions with these variables
 set $target=%1
-set $sdkVersion=%2
-set $runtimeVersion=%3
-set $queue=%4
-set $arch=%5
-set $quarantined=%6
-set $ef=%7
-set $aspnetruntime=%8
-set $aspnetref=%9
-REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
-shift
+set $runtimeVersion=%2
+set $queue=%3
+set $arch=%4
+set $quarantined=%5
+set $ef=%6
+set $aspnetruntime=%7
+set $aspnetref=%8
 set $helixTimeout=%9
+REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
```

Comment:
This comment isn't relevant anymore; no more `shift`

Reply:
I left it for posterity since when we add another argument, we will need to shift to get more still